modules/AlphaWalletFoundation/AlphaWalletFoundation/Transfer/Types/GasLimitWarning.swift

```diff
@@ -10,9 +10,6 @@ import Foundation
 extension TransactionConfigurator {
     public enum GasLimitWarning {
         case tooHighCustomGasLimit
-
```

Comment:
This is strange. Why does it not work?

Reply:
Have no idea, maybe it's because it was defined in foundation pod, different namespaces.